src/ImportExport.ts

```diff
@@ -674,12 +676,7 @@ const addCodeAvailableUses = (): number => {
 }
 
 const addCodeTimeToNextUse = (): number => {
-    let timeInterval = hour
-    timeInterval *= (1 - 0.02 * player.shopUpgrades.calculator4)
-    timeInterval *= (1 - (player.highestSingularityCount >= 125 ? player.highestSingularityCount / 1000 : 0)
-                       - (player.highestSingularityCount >= 200 ? player.highestSingularityCount / 1000 : 0))
-
-    return Math.floor(timeInterval + player.rngCode - Date.now())/1000;
+    return Math.floor(addCodeInterval() + player.rngCode - Date.now())/1000;
```

Comment:
including awWhy's better bug fix, this way there aren't two places to update the interval calculation (which caused the bug initially)

Reply:
Sadly not mine fix, but httpnet one
(I think)

Reply:
you are correct, i think i was talking to you about another issue when I saw his PR and confused the two >_<